apps/examples/src/ExamplePage.tsx

```diff
@@ -39,41 +40,69 @@ export function ExamplePage({
 		return () => cancelAnimationFrame(frame)
 	}, [example])
 
+	const categories = examples.map((e) => e.id)
+
 	return (
 		<div className="example">
 			<div className="example__info">
 				<Link className="example__logo" to="/">
 					<ExamplesTldrawLogo /> examples
 				</Link>
-				<ul className="example__info__list scroll-light" ref={scrollElRef}>
-					{examples
-						.filter((e) => !e.hide)
-						.filter((e) => e.order !== null)
-						.map((e) => (
-							<ListLink
-								key={e.path}
-								ref={e.path === example.path ? activeElRef : undefined}
-								example={e}
-								isActive={e.path === example.path}
-							/>
-						))}
-					<li>
-						<hr />
-					</li>
-					{examples
-						.filter((e) => !e.hide)
-						.filter((e) => e.order === null)
-						.map((e) => (
-							<ListLink
-								key={e.path}
-								ref={e.path === example.path ? activeElRef : undefined}
-								example={e}
-								isActive={e.path === example.path}
-							/>
-						))}
-				</ul>
+				<Accordion.Root
+					type="multiple"
+					defaultValue={categories}
+					className="example__info__list scroll-light"
+					ref={scrollElRef}
+				>
+					{categories.map((c) => (
```

Comment:
nit: (similar to below), it'd be better to have `category` vs `c` - easier to read the code and also easier to grep through when looking for where the variable is used